rllib/core/learner/differentiable_learner_config.py

```diff
@@ -45,3 +60,88 @@ def __post_init__(self):
                 "`learner_class` must be a subclass of `DifferentiableLearner "
                 f"but is {self.learner_class}."
             )
+
+    def build_learner_connector(
```

Comment:
Should we just replace all this duplicate(?) logic with a call to `AlgorithmConfig.build_learner_connector(self, input_observation_space, input_action_space, device)`?

Reply:
This would probably give us a different learner connector pipeline - as this would be the one for the meta learner and not the differentiable learner.